app/routes/post_login.js

```diff
@@ -48,7 +44,7 @@ export default Route.extend({
         this.transitionTo("browse");
       }
     } else {
-      this.transitionTo("account_details");
+      window.location.replace("/account_details");
```

Comment:
@akshaygwd, please test it on all platforms. Transition is required. I would recommend to use `transitionTo` here.